src/client/app/game.service.ts

```diff
@@ -777,4 +778,15 @@ export class GameService {
     });
   }
 
+  public globalLink(target: string, type = 'player') {
+    let name = target;
+    // Remove title from player name
+    if(type === 'player') {
+      const noTitle = target.split(',');
```

Comment:
isn't there a way to pass in just the player name instead of doing this?

Reply:
Sure. I'd need to modify the message interface to store the title

Reply:
I see now. Yeah, I think that's fine. Logic like this will break in the face of someone with a comma in their name (which unfortunately I don't disallow) so I'd rather adjust the code to fit this than fit this to what exists already.

Reply:
Fair enough